app/queues/dns/dns-flow.server.ts

```diff
@@ -11,6 +11,7 @@ import { WorkType } from './dns-record-worker.server';
 import type { FlowJob } from 'bullmq';
 import type { JobRecord } from './dns-record-worker.server';
 import { buildDomain } from '~/utils';
+import logger from '~/lib/logger.server';
```

Comment:
You might have forgotten to remove the import for logger
```suggestion
import logger from '~/lib/logger.server';
```